pkg/routeagent_driver/main.go

```diff
@@ -152,3 +155,23 @@ func annotateNode(clusterCidr []string, cfg *restclient.Config) error {
 
 	return nil
 }
+
+func getUsrForceMssVal(cfg *restclient.Config) int {
+	usrForceMssVal := 0
+
+	k8sClientSet, err := kubernetes.NewForConfig(cfg)
```

Comment:
I think we can directly pass the k8sClientSet as an argument to this API instead of constructing it again here. 
i.e., in the caller of this API, please create the k8sClientSet and use it both for this API as well as for annotateNode() methods.